### PR TITLE
Add March 2023 roundup

### PR DESCRIPTION
**Pull request by saragibby**
Adds the March 2023 round up highlighting Database Connections

### CODE REVIEW

File: www/blog-roundup/2023-03-20-thismonth/index.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+* **March 10** | `geeksforgeeks.org` -  [**How to use Azure Static Web Apps to Deploy an Angular App**](https://www.geeksforgeeks.org/how-to-use-azure-static-web-apps-to-deploy-an-angular-app/) by _Manikandan Shanmugam_ Follow this step by step guide on how you can create and deploy your own Angular app to Azure Static Web Apps with having just a GitHub and an Azure account.
+* **March 20** | `blog.johnnyreilly.com` -  [**Playwright, GitHub Actions and Azure Static Web Apps staging environments**](https://blog.johnnyreilly.com/playwright-github-actions-and-azure-static-web-apps-staging-environments) by _John Reilly_ I was trying to setup a deployment for an SPA application to an Azure Static Web app using TeamCity, but most of the tutorials I found was doing the deployment through CI/CD of Github's Actions .. But what if .. we need to do it on demand from TeamCity, or any other CI/CD tool? You can do this using [the Azure Static Web Apps CLI](https://github.com/Azure/static-web-apps-cli)! Learn how.
```
Comment (Reshmi-Sriram) [inline www/blog-roundup/2023-03-20-thismonth/index.md:71]: I saw John's [article](https://blog.johnnyreilly.com/playwright-github-actions-and-azure-static-web-apps-staging-environments) and it does not look like SWA CLI to me. Maybe the description we've added here is wrong?

File: www/blog-roundup/2023-03-20-thismonth/index.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+---
+
+## Upcoming Events
```
Comment (Reshmi-Sriram) [inline www/blog-roundup/2023-03-20-thismonth/index.md:75]: Do we not wanna call out Student Summit? We can give it more real estate possibly next month, but a mention might be good and timely right?

**Review by Reshmi-Sriram — CHANGES_REQUESTED** [review]
added a couple of comments

File: www/blog-roundup/2023-03-20-thismonth/index.md (hunk 1 above)
Comment (saragibby) [reply www/blog-roundup/2023-03-20-thismonth/index.md:71]: Thanks for catching!

File: www/blog-roundup/2023-03-20-thismonth/index.md (hunk 2 above)
Comment (saragibby) [reply www/blog-roundup/2023-03-20-thismonth/index.md:78]: Thanks for thinking of Summit! I didn't include it initially as some regional events have already completed, but they have the recording already linked up. Getting this added in now 😊

**Pull request merged**